api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -53,6 +53,11 @@ message GrpcService {
 
         // https://grpc.io/grpc/cpp/namespacegrpc.html#a6beb3ac70ff94bd2ebbd89b8f21d1f61
         google.protobuf.Empty google_default = 2;
+
+        // Use local channel credentials. Currently only UDS is supported.
+        // See
+        // https://github.com/grpc/grpc/blob/782e5c2b8dc6973b6fc0959ba08da0f5b9db2f6f/include/grpcpp/security/credentials.h#L239
+        bool local_credentail = 3;
```

Comment:
Shouldn't this be `google.protobuf.Empty`? just like one above. Or define an enum like https://github.com/grpc/grpc/blob/525c59f74cbfe5300ce0c21dbd00fcf7a01e9934/include/grpc/grpc_security_constants.h#L112

Reply:
Changed to google.protobuf.Empty.